mpyg321/mpyg321.py

```diff
@@ -1,6 +1,22 @@
 import pexpect
 from threading import Thread
 
+
+def is_installed(name):
+    """Checks if a program is installed"""
+    import os
+    import subprocess
+
+
+    devnull = open(os.devnull)
+    try:
+        subprocess.Popen([name, "--help"], stdout=devnull, \
```

Comment:
I think we should do this try/except when creating the actual process. Why would we create a process for no reason?

Reply:
Since I pass --help to the command, I don't think it really matters because the process will end almost instantly. That being said, I think your idea is probably better. I will change this run the try/except cycle when creating the actual process rather than creating a dummy process and running the real one later.

Reply:
I've changed the `__init__` method of the `MPyg321Player` class to try to spawn `mpg321` first using `pexpect`, then try the same with `mpg123`, and if both fail then it raises a `FileNotFoundError` stating that a suitable program couldn't be found and that the user should install one of the 2 and try again.